charts/README.md

```diff
@@ -121,6 +121,33 @@ $ helm test $RELEASE_NAME --cleanup
 where `$RELEASE_NAME` is replaced by the name you picked for the Helm release;
 `zenko` in the example above.
 
+Upgrading Your Deployment
+-------------------------
+
+To upgrade an existing Zenko installation to the latest version or simply change deployment
+configuration, you will need to run the following commands. This assumes that your release
+name is `zenko` and that you have previously cloned the repository. 
+
+```shell
+$ cd Zenko/charts
+$ git pull
+$ helm dep build ./zenko
```

Comment:
@ssalaues Maybe use `dependency` instead of `dep` to be more explicit of what the command does?